general/clients/web-config.md

```diff
@@ -0,0 +1,52 @@
+---
+uid: web-config
+title: Jellyfin Web configuration
+---
+
+# Jellyfin Web configuration
+
+## Editing the configuration
+
+The Jellyfin Web default interface can be configured using the `config.json` file in the webroot. Where this is and how to edit it depends on the installation method.
+
+### Debian/Ubuntu/Fedora/CentOS Packages
+
+The configuration can be found at `/usr/share/jellyfin/web/config.json`. This file is registered as a configuration file by the Debian packages, and any changes to the defaults will be handled by `apt` on upgrade.
+
+### Docker
+
+Overriding the default `config.json` can be done with an additional volume parameter to your `docker run` command, e.g.
+
+```sh
+--volume /path/to/config/web-config.json:/jellyfin/jellyfin-web/config.json
```

Comment:
It should be noted that if the target file does not exist, docker will be so helpful as to create a ***directory*** for us.
I'm unfortunately not familiar enough with the markdown flavor here to add that nice warning box.